### ISSUE TITLE
Duplicate folders for import file intellisense suggestions?

### ISSUE THREAD

**Issue by tr1s** [microsoft/vscode#92101]
Is there a reason when I'm importing files that I always get suggested two? I always prefer the one with the trailing slash so I can tab complete to the file I need. Not once have I ever wanted to click the suggestion without the slash. I always have to hit down to the correct one and then tab complete.

Would love to know why this is happening and how I stop it from happening!

<img width="778" alt="CleanShot 2020-03-05 at 12 27 13@2x" src="https://user-images.githubusercontent.com/19172250/76009069-9abc3e00-5ede-11ea-8758-0c2562a2237e.png">

- VSCode Version: Insiders Latest
- OS Version: Latest

Steps to Reproduce:

Import anything at all ever, wether that be in JavaScript or CSS when navigating to a file or image etc. Folders always duplicate, one with a trailing slash and one without.

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by tr1s** [comment]
> extensions disabled

Yes it does. This used to be an extension that handled path intellisense or w/e, but VSCode implemented it at some point awhile ago so this is all default VScode!

<img width="786" alt="CleanShot 2020-03-05 at 15 27 08@2x" src="https://user-images.githubusercontent.com/19172250/76022797-d9a9be00-5ef5-11ea-8d9f-65bf95a3ddc9.png">

Do you not see this when you navigate through paths?

**Comment by mjbvz** [comment]
No.

<img width="1126" alt="Screen Shot 2020-03-05 at 12 38 21 PM" src="https://user-images.githubusercontent.com/12821956/76023568-3e591e80-5ede-11ea-9fce-093ac20760bf.png">


Please provide an example project that demonstrates the issue

**Comment by tr1s** [comment]
This is my mistake.

For anyone who comes across this:

I did actually have [path-autocomplete](https://marketplace.visualstudio.com/items?itemName=ionutvmi.path-autocomplete) installed, but I never reloaded my VSCode when I disabled all my plugins. VSCode is the one without the trailing slashes, which I don't prefer. So all I had to do was disable VSCodes path suggest and then keep the extensions version on.

`"javascript.suggest.paths": false,`
